packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -103,6 +103,7 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
   private _isMultiStackChart: boolean;
   private _tooltipId: string;
   private _highlightedCircleId: string;
+  private _enableComputationOptimization: boolean;
```

Comment:
Add a comment about what this computation optimization is about.

Reply:
Done